hw/dashboard/index.md

```diff
@@ -0,0 +1,39 @@
+# Hardware Development Dashboard
+
+This page serves as the landing spot for all hardware development within the OpenTitan project.
+
+We start off by providing links to the [summary results of various tool-flows](#toolflows-summary-results) run on all of our [Comportable]({{< relref "doc/rm/comportability_specification" >}}) designs.
+This includes DV simulations, FPV and lint, all of which are run with the `dvsim` tool which serves as the common frontend.
+
+The [Comportable designs dashboard](#comportable-designs-dashboard) following it provides links to their design specifications, DV plans and tracks their current stage of development within OpenTitan.
+See the [Hardware Development Stages]({{< relref "/doc/project/hw_stages.md" >}}) for description of the hardware stages and how they are determined.
+
+Next, we focus on all available [processor cores](#processor-cores) and provide links to their design specifications, DV plans and the results from the tool-flows. Finally, we provide the same set of information for all available [top level designs](#top-level-designs), while also including the synthesis results.
+
+## Tool-flows summary results
+
+* [DV simulation summary results, with coverage (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/summary.html)
+* FPV summary results (nightly) (TBD)
+* [Lint summary results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/ascentlint/summary.html)
+
+
+## Comportable designs dashboard
+
+{{< dashboard "hw/ip" >}}
+
+## Processor cores
+
+* `core_ibex`
+  * [User manual](https://ibex-core.readthedocs.io/en/latest)
+  * [DV plan](https://ibex-core.readthedocs.io/en/latest/verification.html)
+  * DV simulation results, with coverage (nightly) (TBD)
+
+## Top level designs
+
+* `top_earlgrey`
+  * [Design specification]({{< relref "hw/top_earlgrey/doc" >}})
+  * DV plan (TBD)
+  * [DV simulation results, with coverage (nightly)](https://reports.opentitan.org/hw/top_earlgrey/dv/latest/results.html)
+  * FPV results (nightly) (TBD)
+  * [Lint results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/lint/latest/results.html)
+  * [Synthesis results (nightly)](https://reports.opentitan.org/hw/top_earlgrey/syn/latest/results.html)
```

Comment:
can you make this TBD and remove the link? 
although this is working, it is not ready to be advertised widely just yet.

CC @sjgitty

Reply:
Done.